language-design.md

```diff
@@ -59,6 +189,11 @@ What does the post on grayscale tell us about the process of API design?
 
 **Response**
 
+It was very interesting to see the process of re-designing the greyscale function and the community feedback.  One of the most things we took away from the post was how different people’s opinions were. Maybe it’s a result of our experiences with other software or simply personal preferences, but what may seem natural to one person is not to another. This is illustrated by the results of the survey: https://docs.google.com/forms/d/e/1FAIpQLScOvDU75W-UNPqDmVScprsO0YT6R_NJr1ZUjllrdNSf3pQPAA/viewanalytics?usp=form_confirm
+
+Both of us agreed with the majority result of the survey as the best option (none of the other options were that intuitive), although we both thought that even this was not a perfect solution because it is not that obvious and may be difficult to debug just from looking at ‘rgb’ values. Even an anonymous survey like this has inherent bias because most of the responses are from people with front-end dev or CSS experience so it is hard to tell which option novices find the most intuitive. 
```

Comment:
The point about which option novices' find most intuitive is quite insightful. Our team really liked the use of a blog to crowdsource people, but being able to get to a blog on web development requires a lot of interest that probably would not come to a novice. Additionally if a novice even found this, the amount of jargon in both the post and comments would be rather daunting. They may be too intimidated to even answer an anonymous poll.